script.js

```diff
@@ -1,29 +1,285 @@
-let blurredAreaModal = document.querySelector('.modal-blurred-area')
 let showInstructionsButton = document.querySelector('#instruction-button')
-let instructionsModal = document.querySelector('.instruction-modal')
 let hideInstructionsModal = document.querySelector('#modal-button')
-let closeModal = (status) => {
-    if(status === true) {
-        blurredAreaModal.style.display = 'none'
-        instructionsModal.style.display = 'none'
-    } else if( status === false) {
-        blurredAreaModal.style.display = 'block'
-        instructionsModal.style.display = 'block'
+
+function openCloseModal(modalOverlay, modalBody, status) {
+    if(status === 'close') {
+        document.querySelector(modalOverlay).style.display = 'none'
+        document.querySelector(modalBody).style.display = 'none'
+    } else if(status === 'open') {
+        document.querySelector(modalOverlay).style.display = 'block'
+        document.querySelector(modalBody).style.display = 'block'
     }
 }
 
 showInstructionsButton.addEventListener('click', (event) => {
     event.stopPropagation()
-    closeModal(false)
+    openCloseModal('.modal-blurred-area', '.instruction-modal', 'open')
 })
 
-hideInstructionsModal.addEventListener('click', (event) => {
-    closeModal(true)
+hideInstructionsModal.addEventListener('click', () => {
+    openCloseModal('.modal-blurred-area', '.instruction-modal', 'close')
 })
 
+let instructionsModal = document.querySelector('.instruction-modal')
+let blurredAreaModal = document.querySelector('.modal-blurred-area')
 window.addEventListener('click', (event) => {
     if (event.target === blurredAreaModal && instructionsModal) {
```

Comment:
what is this condition checking exactly?
- the first side is: `event.target === blurredAreaModal` - why not just put the event listener on the `blurredAreaModal`?
- the second side is: `instructionsModal` - this will never be false because although hidden, the modal does exist in the DOM so `querySelector` earlier will find it